composer.json

```diff
@@ -43,13 +43,13 @@
         "ext-sockets": "*",
         "appwrite/php-clamav": "2.0.*",
         "appwrite/php-runtimes": "0.11.*",
-        "utopia-php/abuse": "0.27.*",
+        "utopia-php/abuse": "0.29.*",
         "utopia-php/analytics": "0.10.*",
-        "utopia-php/audit": "0.29.*",
+        "utopia-php/audit": "0.31.*",
         "utopia-php/cache": "0.8.*",
         "utopia-php/cli": "0.15.*",
         "utopia-php/config": "0.2.*",
-        "utopia-php/database": "0.38.*",
+        "utopia-php/database": "0.40.*",
```

Comment:
we should update the db library here as well right ?

Reply:
done this